src/components/MyForums.js

```diff
@@ -165,26 +233,108 @@ const MyForums = () => {
             </Box>
 
             <Box sx={{ position: 'absolute', top: 150, left: '50%', transform: 'translateX(-50%)' }}>
-            {forums.map((event) => (
-                <Card style={{ width: '800px', marginBottom: '20px' }} key={event.id}>
-                    <CardContent>
-                        <Typography variant="h5" component="div">
-                            {event.forumTitle}<br />
-                        </Typography>
-                        <Typography sx={{ mb: 1.5 }} color="text.secondary">
-                            Posted on {new Date(event.dateTime).toLocaleDateString()}<br />
-                        </Typography>
-                        <Typography sx={{ mb: 1.5 }} color="text.secondary">
-                            <br />{event.description}<br />
-                        </Typography>
-                    </CardContent>
-                </Card>
-            ))}
+                {forums.map((event) => (
```

Comment:
this should be `{forums.map((forum) => {` but Chloe's pr addresses this so this is more for future reference